servant-server/src/Servant/Server/Internal.hs

```diff
@@ -452,3 +474,20 @@ pathIsEmpty = go . pathInfo
 
 ct_wildcard :: B.ByteString
 ct_wildcard = "*" <> "/" <> "*" -- Because CPP
+
+-- * configs
+
+instance HasServer subApi => HasServer (WithNamedConfig name subConfig subApi) where
```

Comment:
Doesn't there need to be an instance for `(WithNamedConfig name config s) :> sublayout` too?